src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const partner = person.sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {person.name}</h2>
+      {person.age && <p className="Person__age">I am {person.age}</p>}
+      <p className="Person__partner">
+        {person.isMarried
+          ? `${person.partnerName} is my ${partner}`
+          : 'I am not married'}
+      </p>
+    </section>
```

Comment:
Also you can use destruction for `person`
```suggestion
  const partner = person.sex === 'm' ? 'wife' : 'husband';
  const { name, age, isMarried } = person;

  return (
    <section className="Person">
      <h2 className="Person__name">My name is {name}</h2>
      
      {age && <p className="Person__age">I am {age}</p>}
      
      <p className="Person__partner">
        {isMarried
          ? `${person.partnerName} is my ${partner}`
          : 'I am not married'}
      </p>
    </section>
```